src/phantom.cpp

```diff
@@ -420,6 +420,12 @@ void Phantom::setProxy(const QString& ip, const qint64& port, const QString& pro
     }
 }
 
+int Phantom::remoteDebugPort() const
+{
+    QApplication::processEvents();
```

Comment:
I agree. I use this to make sure that this will work:
```cpp
obj->setProperty("_q_webInspectorServerPort", inspectorServer->listen(port.toInt()));
```
because ```setProperty``` rises event internally and that event then sets up corresponding property.